src/modules/sipt/sipt.c

```diff
@@ -102,10 +115,30 @@ static sipt_header_map_t sipt_header_mapping[] =
 		{{"CHARGE_INDICATOR", 1}, 
 			{NULL, 0}
 		}},
-        {"REDIRECTION_INFO", ISUP_PARM_REDIRECTION_INFO, 
-                {{NULL, 0}}  },
-        {"REDIRECTION_NUMBER", ISUP_PARM_REDIRECTION_NUMBER, 
-            		{{"NATURE_OF_ADDRESS", 1}, 
+	{"REDIRECTION_INFO", ISUP_PARM_DIVERSION_INFORMATION,
```

Comment:
wouldn't it be better to keep "REDIRECTION_INFO" as is
and add
"DIVERSION_INFO" as an extra parameter?

Reply:
@tsearle It seems to be just a refactoring, have a look to the line 904 of the sipt.c diff/patch. It still returns the same function.

Reply:
This existing "REDIRECTION_INFO" actually gets information from diversion info (ISUP_PARM_DIVERSION_INFORMATION) and not from redirection info header (ISUP_PARM_REDIRECTION_INFO).
So this old name is incorrect. It would be better to rename the old "REDIRECTION_INFO" to "DIVERSION_INFO". 
But it will break existing behavior. If someone uses this old "REDIRECTION_INFO" he will get wrong results.
So I did not change existing behavior, only corrected documentation of "REDIRECTION_INFO"